sdk/ai/azure-ai-generative/tests/synthetic_qa/data/generated_qa_pf_conv.jsonl

```diff
@@ -0,0 +1,5 @@
+{"chat_history":"[]","question":"What is Compute Instance?","ground_truth":"Compute instance is ..."}
```

Comment:
chat history is not supposed to be a string

Reply:
fixed in latest commit